dev-client/src/components/projects/CreateProjectView/InnerForm.tsx

```diff
@@ -0,0 +1,82 @@
+import {HStack, Input, Text, VStack} from 'native-base';
+import {Formik, FormikConfig, FormikProps} from 'formik';
+import RadioBlock from '../../common/RadioBlock';
+import validationSchema from './validation';
+import {IconButton} from '../../common/Icons';
+import {useTranslation} from 'react-i18next';
+import SaveFAB from '../../common/SaveFAB';
+import ErrorMessage from '../../common/ErrorMessage';
+
+export interface FormValues {
+  name: string;
+  description: string;
+  privacy: 'PUBLIC' | 'PRIVATE';
+}
+
+interface Props {
+  onSubmit?: FormikConfig<FormValues>['onSubmit'];
+}
+
+export default function InnerForm({
+  onSubmit = values => console.debug(values),
+}: Props) {
+  const {t} = useTranslation();
+  return (
+    <Formik
+      initialValues={{name: '', description: '', privacy: 'PRIVATE'}}
+      onSubmit={onSubmit}
+      validationSchema={validationSchema}>
+      {({
+        handleChange,
+        handleBlur,
+        handleSubmit,
+        values,
+        isSubmitting,
+      }: FormikProps<FormValues>) => {
```

Comment:
question: ok now i am wondering a bit more what the performance implications of inline function components are lol. i'm not against this, but also the level of line nesting gets larger when the inline component is larger and i find it a bit harder to parse